services/common/src/components/forms/BaseInput.tsx

```diff
@@ -84,16 +84,21 @@ interface BaseViewInputProps {
   label?: string | ReactNode;
   value: string | number;
   showNA?: boolean;
+  displayOverride?: ReactNode;
 }
-export const BaseViewInput: FC<BaseViewInputProps> = ({ label = "", value = "", showNA = true }) => {
+export const BaseViewInput: FC<BaseViewInputProps> = ({ label = "", value = "", showNA = true, displayOverride }) => {
   const hasValue = value !== "";
   const displayValue = hasValue ? value.toString() : EMPTY_FIELD;
+
+  const valueElement = displayOverride && hasValue
```

Comment:
I was just annoyed wanting to show a list of values and wanted to be able to customize that to not be a paragraph. Only change here.